pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/db/dao/mercury/ClinicalOrderDaoTest.java

```diff
@@ -0,0 +1,267 @@
+package org.broadinstitute.dsm.db.dao.mercury;
+
+import liquibase.pro.packaged.M;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.DbTxnBaseTest;
+import org.broadinstitute.dsm.db.MedicalRecord;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.SmId;
+import org.broadinstitute.dsm.db.Tissue;
+import org.broadinstitute.dsm.db.dao.ddp.institution.DDPInstitutionDao;
+import org.broadinstitute.dsm.db.dao.ddp.medical.records.MedicalRecordDao;
+import org.broadinstitute.dsm.db.dao.ddp.onchistory.OncHistoryDao;
+import org.broadinstitute.dsm.db.dao.ddp.participant.ParticipantDao;
+import org.broadinstitute.dsm.db.dao.ddp.tissue.TissueSMIDDao;
+import org.broadinstitute.dsm.db.dao.kit.KitTypeDao;
+import org.broadinstitute.dsm.db.dao.kit.KitTypeImpl;
+import org.broadinstitute.dsm.db.dto.ddp.institution.DDPInstitutionDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.db.dto.kit.KitTypeDto;
+import org.broadinstitute.dsm.db.dto.mercury.ClinicalOrderDto;
+import org.broadinstitute.dsm.db.dto.mercury.MercuryOrderDto;
+import org.broadinstitute.dsm.db.dto.onchistory.OncHistoryDto;
+import org.broadinstitute.dsm.juniperkits.TestKitUtil;
+import org.broadinstitute.dsm.service.admin.UserAdminTestUtil;
+import org.broadinstitute.dsm.util.MedicalRecordUtil;
+import org.broadinstitute.lddp.util.JsonTransformer;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.util.*;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+import static org.broadinstitute.dsm.service.admin.UserAdminService.USER_ADMIN_ROLE;
+import static org.broadinstitute.dsm.statics.DBConstants.KIT_SHIPPING;
+import static org.broadinstitute.dsm.statics.DBConstants.PT_LIST_VIEW;
+
+@Slf4j
+public class ClinicalOrderDaoTest extends DbTxnBaseTest {
+
+    private static final String TEST_USER = "testuser";
+
+    private static final MedicalRecordDao mrDao = new MedicalRecordDao();
+
+    private static final OncHistoryDao oncHistoryDao = new OncHistoryDao();
+
+    private static final DDPInstitutionDao institutionDao = new DDPInstitutionDao();
+
+    private static final Tissue tissueDao = new Tissue();
+
+    private static int medicalRecordId = -1;
+
+    private static final MercuryOrderDao mercuryOrderDao = new MercuryOrderDao();
+
+    private static final ClinicalOrderDao clinicalOrderDao = new ClinicalOrderDao();
```

Comment:
I'm not proud of the messiness of the test setup/teardown.  I would like to clean this up so that others can make use of utility classes for samples/tissues/onchistory/mr.  I would like to break that work out to a separate PR so that we can get this DAO onto the mainline for integration testing.